apps/openmw/mwsound/soundmanagerimp.cpp

```diff
@@ -368,22 +366,39 @@ namespace MWSound
 
             mMusicFiles[mCurrentPlaylist] = filelist;
 
+            // Build mMusicToPlay for this playlist
+            std::vector<int> temp;
+            temp.reserve(filelist.size());
+            for(int it = 0; it < filelist.size(); it++)
```

Comment:
GCC complains about "comparison between signed and unsigned" here. IMO "it" should be unsigned int.